crates/iota-open-rpc/spec/openrpc.json

```diff
@@ -1301,26 +1301,26 @@
             "name": "Result",
             "value": {
               "minSupportedProtocolVersion": "1",
-              "maxSupportedProtocolVersion": "5",
-              "protocolVersion": "1",
+              "maxSupportedProtocolVersion": "6",
+              "protocolVersion": "6",
               "featureFlags": {
                 "accept_zklogin_in_multisig": false,
                 "bridge": false,
-                "convert_type_argument_error": false,
+                "convert_type_argument_error": true,
```

Comment:
I wonder why so many values in this file change, is that correct?

Reply:
Yes, seems they were not properly re-generated for v5, which introduced all these changes